src/options/options.ts

```diff
@@ -129,12 +129,19 @@ export async function getOptions({
 async function getRequiredOptions(combined: OptionsFromConfigAndCli) {
   const { accessToken, repoName, repoOwner, globalConfigFile } = combined;
 
-  if (accessToken && repoName && repoOwner) {
-    return { accessToken, repoName, repoOwner };
+  const resolvedAccessToken =
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    accessToken?.trim() ||
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    process.env.BACKPORT_ACCESS_TOKEN?.trim() ||
+    undefined;
```

Comment:
Rename to `trimmedAccessToken`. "resolved" makes it sounds like an async value

```suggestion
  const trimmedAccessToken =
    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
    accessToken?.trim() || process.env.BACKPORT_ACCESS_TOKEN?.trim();
```